apis/sesv2/v1alpha1/zz_email_identity.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2021 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by ack-generate. DO NOT EDIT.
+
+package v1alpha1
+
+import (
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+// EmailIdentityParameters defines the desired state of EmailIdentity
+type EmailIdentityParameters struct {
+	// Region is which region the EmailIdentity will be created.
+	// +kubebuilder:validation:Required
+	Region string `json:"region"`
+	// The configuration set to use by default when sending from this identity.
+	// Note that any configuration set defined in the email sending request takes
+	// precedence.
+	ConfigurationSetName *string `json:"configurationSetName,omitempty"`
```

Comment:
I think we can have a [referencer](https://github.com/crossplane/provider-aws/blob/master/CODE_GENERATION.md#referencer-fields) here for `ConfigurationSet` type.